app/controllers/statement_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class StatementController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @statement = Statement.find_or_initialize_by(
```

Comment:
A comment explaining why you're looking for an already created statement would be helpful